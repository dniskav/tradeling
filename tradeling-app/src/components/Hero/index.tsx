import React from 'react';
import Logo from '../Logo/index';
import { useTranslation } from 'react-i18next';
import styled from 'styled-components';

const Container = styled.div`
  display: flex;
  & > .title {
    display: flex;
    flex-direction: column;
    text-align: left;
    h1 {
      margin: 2px 0 0 0;
      font-size: 20px;
      letter-spacing: 0px;
    }
    h2 {
      font-size: 14px;
      font-weight: 400;
      letter-spacing: 0.35px;
      margin: 0;
    }
  }
`;

const Hero: React.FC = () => {
  const { t } = useTranslation();
  return (
    <Container>
      <Logo />
      <div className="title">
        <h1>{t('Hero.title')}</h1>
        <h2>{t('Hero.subtitle')}</h2>
      </div>
    </Container>
  );
};

export default Hero;
