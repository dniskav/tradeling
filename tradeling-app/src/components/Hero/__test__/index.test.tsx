import Hero from "../index";
import { shallow } from "enzyme";

describe("<Hero />", () => {
  const tree = shallow(<Hero />);
  it("Hero should match snapshot", () => {
    expect(tree).toMatchSnapshot();
  });
});
