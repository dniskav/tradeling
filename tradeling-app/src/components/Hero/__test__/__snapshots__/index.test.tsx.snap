// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> Hero should match snapshot 1`] = `
<styled.div>
  <Logo />
  <div
    className="title"
  >
    <h1>
      Hero.title
    </h1>
    <h2>
      Hero.subtitle
    </h2>
  </div>
</styled.div>
`;
